### PR TITLE
Update zendoo badge link

### PR DESCRIPTION
**Pull request by max-sixty**
It currently isn't a badge and the link is broken?

### CODE REVIEW

**Comment by max-sixty** [comment]
image

File: README.md [hunk 1]
```diff
@@ -46,15 +45,15 @@ provide a powerful and concise interface. For example:
 
 - Apply operations over dimensions by name: `x.sum('time')`.
 - Select values by label instead of integer location:
-    `x.loc['2014-01-01']` or `x.sel(time='2014-01-01')`.
```
Comment (max-sixty) [inline README.md:49]: These are autoformatting, can revert if anyone has a preference

File: README.md [hunk 2]
```diff
@@ -7,7 +7,6 @@
 [![Available on pypi](https://img.shields.io/pypi/v/xarray.svg)](https://pypi.python.org/pypi/xarray/)
 [![Formatted with black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
 [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
```
Comment (dcherian) [inline README.md:9]:
```suggestion
[![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.11183201.svg)](https://doi.org/10.5281/zenodo.11183201)
```

Here's the latest one from Zenodo

**Comment by dcherian** [comment]
Works for me right now btw:
image

**Comment by max-sixty** [comment]
Great, thanks for the change, I'll repurpose this PR to update.

I guess I looked when the link was down, must have been a one-off...

**Pull request merged**